mne_bids/tests/test_copyfiles.py

```diff
@@ -50,27 +49,26 @@ def test_get_brainvision_encoding():
         f.readlines()
 
 
-def test_get_brainvision_paths():
+def test_get_brainvision_paths(tmpdir):
     """Test getting the file links from a BrainVision header."""
-    test_dir = _TempDir()
     data_path = op.join(base_path, 'brainvision', 'tests', 'data')
     raw_fname = op.join(data_path, 'test.vhdr')
 
     with pytest.raises(ValueError):
         _get_brainvision_paths(op.join(data_path, 'test.eeg'))
 
     # Write some temporary test files
-    with open(op.join(test_dir, 'test1.vhdr'), 'w') as f:
+    with open(tmpdir / 'test1.vhdr', 'w') as f:
```

Comment:
You can actually use the OOP approach offered through Path here

```suggestion
    with (tmpdir / 'test1.vhdr').open('w') as f:
```

But I guess it's a matter of taste

Reply:
good to know :+1: happy to accept the suggestion if there is a benefit. As of taste, I am not sure what I like better.

Reply:
Let's just leave it the wait it currently is then :)